doc/wg/documentation/README.md

```diff
@@ -0,0 +1,30 @@
+Tock Documentation Working Group (DOC)
+======================================
+
+- Working Group Charter
+- Adopted xx/xx/xxxx
+
+## Goals
+
+The goals of the Tock Documentation Working Group (DOC) are to:
+
+- maintain and improve the [Tock Book][book]
+- oversee and improve the [API docs][api]
+- maintain and improve the informational content on the [TockOS website][tockos]
+- review changes to TockOS documentation in the [doc folder][doc]
+
+## Members
+
+- Brad Campbell (Chair), University of Virginia
+
+## Code Purview
+
+The DOC working group maintains:
+
+- https://github.com/tock/book
+- https://github.com/tock/tock/tree/master/doc
```

Comment:
Should this exclude normative documentation like TRDs? I don't foresee this being a problem in practice, but this might create a conflict between purview of those between Core and DOC WGs.

Reply:
Adding to this, that last folder also contains documentation of other working groups (such as call notes). I'm not sure whether the burden of maintaining that should be placed on a DOC WG as well.

Reply:
> Should this exclude normative documentation like TRDs?

Yes I think so.

> contains documentation of other working groups (such as call notes). I'm not sure whether the burden of maintaining that should be placed on a DOC WG as well.

I agree this should be more narrowly scoped.